test/boost/s3_test.cc

```diff
@@ -20,13 +20,13 @@
 
 /*
  * Tests below expect minio server to be running on localhost
- * with the bucket named "testbucket" created with unrestricted
- * anonymous read-write access
+ * with the bucket named env['S3_PUBLIC_BUCKET_FOR_TEST'] created with
+ * unrestricted anonymous read-write access
  */
 
 SEASTAR_THREAD_TEST_CASE(test_client_put_get_object) {
     const ipv4_addr s3_server(::getenv("MINIO_SERVER_ADDRESS"), 9000);
-    const sstring name(fmt::format("/testbucket/testobject-{}", ::getpid()));
+    const sstring name(fmt::format("/{}/testobject-{}", ::getenv("S3_PUBLIC_BUCKET_FOR_TEST"), ::getpid()));
```

Comment:
What will happen if somebody attempts to run this without `test.py`?
There should be an error about the unset variable, instead of a cryptic error because of the bucket was null.

Reply:
Fair point. Fixed in v2